scheduler/reconcile_test.go

```diff
@@ -5464,6 +5464,9 @@ func TestReconciler_Disconnected_Client(t *testing.T) {
 					},
 				},
 			},
+			reconcileStrategy: structs.ReconcileOptionBestScore,
+			callPicker:        true,
+			pickResult:        "original",
```

Comment:
Actually going to remove this, as the name of the test is `stop-original-alloc....`